src/main/java/com/wiley/driver/factory/capabilities/ChromeCaps.java

```diff
@@ -26,17 +26,18 @@ public ChromeCaps(DesiredCapabilities customCaps, UnexpectedAlertBehaviour alert
         this.platform = platform;
     }
 
+    @SuppressWarnings("unchecked") // Just in case getCapability return value type changes in future versions
```

Comment:
I don't seem to understand what is the purpose of the suppressed warning? If the return type is changes - we won't to see the issue highlighted by IDE/Sonar/etc. What is the benefit of ignoring it?

Reply:
It is done intentionally to ignore the warning, because I'm forcing certain data types without checks by design.